packages/appium/lib/appium.js

```diff
@@ -516,6 +524,17 @@ class AppiumDriver extends BaseDriver {
     // handling the command and which didn't
     plugins.length && this.logPluginHandlerReport({cmd, cmdHandledBy});
 
+    // And finally, if the command was createSession, we want to migrate any plugins which were
+    // previously sessionless to use the new sessionId, so that plugins can share state between
+    // their createSession method and other instance methods
+    if (cmd === 'createSession' && this.sessionlessPlugins.length && !res.error) {
```

Comment:
I might want to create a constant for this string

Reply:
yeah, it was already constantified in base driver so i'll just hook up the import/export pipes.